src/EntityRepository/CustomerRepository.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\EntityRepository;
```

Comment:
Repository is having not standard repository directory which also does not make app framework agnostic, but also may have some changes to make it clearly Symfony app